src/Symfony/Component/Cache/Adapter/ApcuAdapter.php

```diff
@@ -11,17 +11,109 @@
 
 namespace Symfony\Component\Cache\Adapter;
 
-use Symfony\Component\Cache\Traits\ApcuTrait;
+use Symfony\Component\Cache\CacheItem;
+use Symfony\Component\Cache\Exception\CacheException;
 
 class ApcuAdapter extends AbstractAdapter
 {
-    use ApcuTrait;
+    public static function isSupported()
+    {
+        return \function_exists('apcu_fetch') && filter_var(ini_get('apc.enabled'), FILTER_VALIDATE_BOOLEAN);
+    }
 
     /**
      * @throws CacheException if APCu is not enabled
      */
     public function __construct(string $namespace = '', int $defaultLifetime = 0, string $version = null)
     {
-        $this->init($namespace, $defaultLifetime, $version);
+        if (!static::isSupported()) {
+            throw new CacheException('APCu is not enabled');
+        }
+        if ('cli' === \PHP_SAPI) {
```

Comment:
what about phpdbg ?

Reply:
That's just copy-pasting from the trait, let's not change nor wonder here.